File: Results/DER/Translation/translation_output/misleading_test/codeqwen/codenet/python/java/p03011.java
Here is the Java version of the provided Python code:

```java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        int c = scanner.nextInt();
        System.out.println(a + b + c - Arrays.stream(new int[]{a, b, c}).max().getAsInt());
    }
}
```

This Java code does the same thing as the Python code. It reads three integers from the user, adds them together, and then subtracts the maximum of the three integers from the sum. The `Arrays.stream(new int[]{a, b, c}).max().getAsInt()` part of the code does the same thing as the `max(a,b,c)` part of the Python code. It creates a stream of the three integers, finds the maximum value, and then returns it as an integer.

File: Results/DER/Translation/translation_output/misleading_test/codeqwen/codenet/java/python/p03011.py
Here is the Python equivalent of the Java code:

```python
def main():
    P, Q, R = map(int, input().split())
    max_val = 0

    if P >= Q:
        if P >= R:
            max_val = Q + R
        else:
            max_val = P + Q
    elif Q >= R:
        if Q >= P:
            max_val = P + R
        else:
            max_val = Q + R
    elif R >= P:
        if R >= Q:
            max_val = P + Q
        else:
            max_val = P + R

    print(max_val)

if __name__ == "__main__":
    main()
```

This Python code does the same thing as the Java code. It reads three integers from the user, and then determines the maximum value by comparing the three numbers. The maximum value is then printed to the console.
